example/src/index.js

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import ThemeLoader from './ThemeLoader';
+
+
+import '@textkernel/oneui/dist/oneui.css';
```

Comment:
Is it also possible to import the source in your own css and compile it as part of the application build process? Or can you only use the precompiled version like this?

Reply:
But that's exactly what is demonstrated here, it's it?
`example/src/index.js` is your application and you just importing `'@textkernel/oneui/dist/oneui.css';` in it.

Reply:
What I mean is your application has a Scss of its own and you import the OneUI scss in it: `@import '@textkernel/oneui/src/themes/oneui/oneui.scss'`. So everything (your application's CSS + OneUI CSS) can be bundled into 1.

Reply:
I really think that importing sass from OneUI is something we shouldn't encourage/support.
Otherwise any variable we have in sass becomes oneUI's "API" and therefore we should support it and maintain backwards compatibility etc.

Reply:
As long as we expose the `vars` to the user, I don't see any use case for this scenario. Might be good indeed to consider it later on when gets necessary.

Reply:
Ok, if you have arguments against it, it's fine -- just wanted to make sure it was considered